instrumentation-api-incubator/src/main/jflex/SqlSanitizer.jflex

```diff
@@ -273,6 +308,33 @@ WHITESPACE          = [ \t\r\n]+
     }
   }
 
+  private class Create extends CompositeOp {
+    boolean handleIdentifier() {
```

Comment:
looks similar to the cases below - maybe extract into common base class

Reply:
Updated.